tests/integration_tests/performance/test_network_latency.py

```diff
@@ -32,33 +32,39 @@
 
 def pass_criteria():
     """Define pass criteria for the statistics."""
-    delta = BASELINES[platform.machine()]["delta"]
-    target = BASELINES[platform.machine()]["target"]
+    delta = LATENCY_AVG_BASELINES[platform.machine()]["delta"]
+    target = LATENCY_AVG_BASELINES[platform.machine()]["target"]
 
     return {
-        function.Avg.name(): criteria.EqualWith(target, delta)
+        function.Avg.__name__: criteria.EqualWith(LATENCY_AVG_BASELINES[
+                                                      platform.machine()])
     }
 
 
 def measurements():
     """Define the produced measurements."""
-    return [types.MeasurementDef(LATENCY, "millisecond"),
+    return [types.MeasurementDef(LATENCY, "ms"),
             types.MeasurementDef(PKT_LOSS, "percentage")]
 
 
 def stats():
     """Define statistics based on the measurements."""
     # Add default statistics for "latency" measurement.
-    stats_defs = types.StatisticDef.defaults(LATENCY,
-                                             [function.Min, function.Stddev,
-                                              function.Max, function.Avg,
-                                              function.Percentile50,
-                                              function.Percentile90,
-                                              function.Percentile99],
-                                             pass_criteria())
-    stats_defs.append(consumer.StatisticDef(PKT_LOSS_STAT_KEY,
-                                            PKT_LOSS,
-                                            function.GetFirstObservation))
+    stats_defs = types.StatisticDef.defaults(
+        LATENCY,
+        # We are going to consume multiple statistics
+        [function.ValuePlaceholder(function.Avg.__name__),
```

Comment:
seems rather verbose. the `name()` method seems to still be available.
Moreover, tying this test to the `function.*` functions can be misleading. The only thing it uses from that classes are the names. I'd say just provide local names

Reply:
Done. Please take a look.

The `name()` method is an object one and it is mainly relevant post definition, when for a specific statistic definition we want to get its assigned name. Previously, the method was thought of as a class method, when the functions were lazily instantiated. I've changed this aspect to remove redundant information and for separation of concerns - the functions are the sole owner of their processing identification and this identification is used by statistics as their name.